### ISSUE TITLE
Refactoring: Extract constants from Controllers

### ISSUE THREAD

**Issue by Aivean** [spaceshelter/orbitar#105]
Currently in Controllers (`InviteController.ts`, `UserController.ts`, etc) there are constant error codes, that are sometimes reused. 

For example, see discussion here:
https://github.com/spaceshelter/orbitar/pull/104/files/5a794da991a7f2613ae6ff1b393f7380246c0ef3#diff-68da325e482b3f5889e8e22074b791699eb4d2d310148be6ae9ff0c1c571845c

It would make sense to extract such constants at least to the file level, like this:
```ts
const ERROR_CODES = {
   INVALID_CODE: 'invalid-code',
   INVITE_NOT_FOUND: 'invite-not-found',
   ...
}
```

**Comment by Aivean** [comment]
Hi @Alao-Abiodun, thank you for showing interest in the project. Do you have any specific questions regarding this issue? As the description states, it's pretty much about replacing repeating strings with constants.

**Comment by Alao-Abiodun** [comment]
Not really so regarding the issue, what I understand is that when I check the `UserController.ts` and `InviteController.ts` files I found the string 'invalid code' that is been called repeatedly. So to solve it we pretty much create constant variables and invoke the variables into the space where the string will be used. I don't know If I am correct.

**Comment by Aivean** [comment]
Yep, sounds like you get the gist of it.

**Comment by Alao-Abiodun** [comment]
Ok. Thanks, I will start something on it now.

**Comment by Alao-Abiodun** [comment]
Hi @Aivean I ran into some error when I was doing some setup on the backend folder. Here is the screenshot. 
<img width="578" alt="image" src="https://user-images.githubusercontent.com/46402042/186292633-e0168852-fe6c-4811-a8a4-d3b7d07f2a69.png">

**Comment by Aivean** [comment]
It appears that our [scripts](https://github.com/spaceshelter/orbitar/blob/dev/backend/package.json#L9), specifically `NODE_ENV=development node migrations.js` doesn't work on Windows. 

I found some details about [that issue here](https://stackoverflow.com/a/11928210/1349366) (TODO: document it).

You can try to work around that by changing the scripts locally to  simply `node migrations.js` and setting `NODE_ENV` externally for the shell session, like this: `SET NODE_ENV=development`.

**Comment by Alao-Abiodun** [comment]
> It appears that our [scripts](https://github.com/spaceshelter/orbitar/blob/dev/backend/package.json#L9), specifically `NODE_ENV=development node migrations.js` doesn't work on Windows.
> 
> I found some details about [that issue here](https://stackoverflow.com/a/11928210/1349366) (TODO: document it).
> 
> You can try to work around that by changing the scripts locally to simply `node migrations.js` and setting `NODE_ENV` externally for the shell session, like this: `SET NODE_ENV=development`.

Ok, Thanks I will try that out

**Comment by Alao-Abiodun** [comment]
It worked. So I tried the build and start the server but it keeps breaking without starting the server.
<img width="585" alt="image" src="https://user-images.githubusercontent.com/46402042/186298925-b5ff0b7b-97bc-4833-ab68-84fe59e850be.png">

**Comment by Aivean** [comment]
I'm sorry, I don't see any errors in your screenshot, please try to find the logs.

**Comment by Alao-Abiodun** [comment]
What I mean is that when I start the server, I am supposed to see a logger result of `HTTP server is listening on 5001` but nothing is showing on terminal, it just break off and there is no error to show for it as well.

**Comment by Aivean** [comment]
Fixed by #122
